kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerControllerImpl.java

```diff
@@ -221,7 +222,15 @@ public synchronized KieServerSetup connect(KieServerInfo serverInfo) {
         return serverSetup;
     }
 
+    private boolean isOpenShiftSupported() {
```

Comment:
please change this for a more generice flag naming like disable server setup validation or the sort.
bringing the environmente here is no good

Reply:
After a quick discussion, @mswiderski and @elguardian are fine with this approach. @cristianonicolai could help having a quick check and get it merged. Thanks!